impl/maven-cli/src/main/java/org/apache/maven/cling/invoker/BaseParser.java

```diff
@@ -526,23 +527,37 @@ protected List<CoreExtension> readCoreExtensionsDescriptorFromFile(Path extensio
     }
 
     protected List<CoreExtension> validateCoreExtensionsDescriptorFromFile(
-            Path extensionFile, List<CoreExtension> coreExtensions) {
+            Path extensionFile, List<CoreExtension> coreExtensions, boolean allowMetaVersions) {
         Map<String, List<InputLocation>> gasLocations = new HashMap<>();
+        Map<String, List<InputLocation>> metaVersionLocations = new HashMap<>();
         for (CoreExtension coreExtension : coreExtensions) {
             String ga = coreExtension.getGroupId() + ":" + coreExtension.getArtifactId();
             InputLocation location = coreExtension.getLocation("");
             gasLocations.computeIfAbsent(ga, k -> new ArrayList<>()).add(location);
+            // TODO: metaversions could be extensible enum with these two values out of the box
```

Comment:
@gnodet maybe not extensible enum right now, but where would be their proper place? Maybe not even in API (to draw attention) for start, just some constants somewhere?